lua/orgmode/api/init.lua

```diff
@@ -99,4 +105,54 @@ function OrgApi.refile(opts)
   return true
 end
 
+--- Insert a link to a given location at the current cursor position
+--- @param link_location string
+--- @return boolean
+function OrgApi.insert_link(link_location)
```

Comment:
Let's abstract all this logic in `Hyperlinks` and then require that one here. I'm trying not to depend on `orgmode.api` in the core, but vice versa.

Reply:
I have to admit, that I just looked at other functions, which had similar calling schemes, but I agree, that it would be cleaner to move them into `Hyperlinks`.

Reply:
I moved the code as you suggested.